### PR TITLE
Added the footer

### LINKED ISSUE #306: Footer for the website.

### PR DESCRIPTION
**Pull request by sid22**
I have used the red,white logo without the text here, the main logo with the Bugheist text can also be used here.
Any suggestions for the same?

Fixes #306

### CODE REVIEW

File: website/templates/base.html [hunk 1]
```diff
@@ -101,9 +101,6 @@
                     <li style="float:left;">
                         <a href="#" id="startTour" class="start-tour">Tour</a>
                     </li>
-                    <li style="float:left;">
-                        <a href="/search" class="sponsor-hunt">Search</a>
```
Comment (seanauriti) [inline website/templates/base.html]: Please keep this until it's added.

File: website/templates/base.html [hunk 2]
```diff
@@ -236,15 +270,6 @@
                   $.notify(message,{ position: 'top center', className: "" }); 
                 }
             </script>
-            <script>
```
Comment (seanauriti) [inline website/templates/base.html]: Please keep this.

**Review by fredfalcon — CHANGES_REQUESTED** [review]

File: website/templates/base.html (hunk 1 above)
Comment (goyal-sidd) [inline website/templates/base.html]: Sure.

**Comment by goyal-sidd** [comment]
Is the logo correct? Should I use the Bugheist one (the one with text) or this looks fine?

**Review by fredfalcon — APPROVED** [review]

**Comment by seanauriti** [comment]
Thanks!

**Comment by seanauriti** [comment]
Interested to see what it looks like with the text logo...

**Pull request merged**